### ISSUE TITLE
[FR]: Validator Alerts in Alert Box

### ISSUE THREAD

**Issue by daGscheid** [stereum-dev/ethereum-node#1445]
### Describe your idea

Extend the Alert Box functionality by including chain duties. 

I propose the following alerts & notifications for that: 

Color | Category | Extra Text | "Name" |  Value Displayed | Icon | Condition | Click
--- | --- | --- | --- | --- | --- | --- | ---
🟢 | VALIDATOR | Congratulations | First Attestation | "Epoch Number" | <img src="https://user-images.githubusercontent.com/82385103/213939918-20621031-caef-4a5e-976d-9e1c7b3bb2ff.png" width="50" height="50"/> | Condition: First Attestation after New Setup   | 
🔴 | VALIDATOR  | CRITICAL WARNING| Attestation Missed | "Epoch Number" | <img src="https://user-images.githubusercontent.com/82385103/213939918-20621031-caef-4a5e-976d-9e1c7b3bb2ff.png" width="50" height="50"/>  | Condition: A key missed an attestation  |
🟢 | VALIDATOR  | Congratulations | Block Produced | "Epoch Number" | | Condition: Your node got to successfully produce a block   |
🔴 | VALIDATOR  | CRITICAL WARNING | Block Missed | "Epoch Number" | | Condition: A key missed an attestation  |
🟢 | VALIDATOR  | Congratulations | Start Sync Committee Duty | "Epoch Number" | | Condition: Became part of Sync Committee  | 
🟢 | VALIDATOR  | Count the rewards | End Sync Committee Duty | "Epoch Number" |  | Condition: Stopped being part of Sync Committee  | 
🔴 | VALIDATOR  | CRITICAL WARNING | Missed Sync Committee Duty | "Epoch Number" |  |  Condition: Missed Sync Committee Duty during any Epoch | 

### Considered Alternatives?

--

### Additional Context

Basically build up of the current script we already using in Staking Page. If done correctly we should also discuss how we could use this for extended validator stats! 

### Acknowledgment

- [X] I looked through the feature requests/enhancements planned for this project before submitting a new idea
